src/LibSDL2.jl

```diff
@@ -12,6 +12,9 @@ export SDL2_image_jll
 using SDL2_ttf_jll
 export SDL2_ttf_jll
 
+using SDL2_gfx_jll
+export SDL2_gfx_jll
```

Comment:
This can be done automatically by adding `SDL2_gfx_jll` here in the generator config: https://github.com/JuliaMultimedia/SimpleDirectMediaLayer.jl/blob/master/gen/generator.toml#L7

Docs: https://github.com/JuliaInterop/Clang.jl/blob/master/gen/generator.toml#L34